core/templates/pages/community-dashboard-page/modal-templates/question-suggestion-review-modal.controller.spec.ts

```diff
@@ -146,13 +147,15 @@ describe('Question Suggestion Review Modal Controller', function() {
       expect($scope.skillRubricExplanations).toEqual(['explanation']);
     });
 
-    it('should define validation error as null', function() {
-      expect($scope.validationError).toBe(undefined);
-      $scope.questionChanged();
-      expect($scope.validationError).toBe(null);
-    });
+    it('should define validation error as null when question changes',
```

Comment:
I mostly don't understand the point of this test. What is the broader implication of the validationError being null?

Is it more like "question exists" vs "question didn't exist before", rather than "question changed"? I don't see the point of one question corresponding to "undefined" and the other corresponding to "null".

Reply:
Done.